app/app.go

```diff
@@ -697,7 +698,19 @@ func New(
 	app.UpgradeKeeper.SetUpgradeHandler(plan0_7_0, func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 		// The default genesis parameters of `feemarket` module are fine, because the `InitialBaseFee (1000000000)` is much lower than the current minimal gas price,
 		// so it don't have effect at the beginning, we can always adjust the parameters through the governance later.
-		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
+		updatedVM, err := app.mm.RunMigrations(ctx, app.configurator, fromVM)
+		if err != nil {
+			return nil, err
+		}
+		// Override feemarket parameters
+		app.FeeMarketKeeper.SetParams(ctx, feemarkettypes.Params{
+			NoBaseFee:                true, // enable later through gov proposal
```

Comment:
another solution couldnt be to set `london_block` at the upgrade height?

Reply:
do you mean change parameter in upgrade handler, or open another parameter change gov proposal?

Reply:
change parameter in upgrade handler. 

If I understood correctly, the problem is that the base fee is activated while the london hardfork is not?

Reply:
we might need to set the fee market parameters together too?

Reply:
yes we will need to update the handler with the correct feemarket parameters